ArduPlane/tailsitter.cpp

```diff
@@ -438,8 +442,10 @@ void Tailsitter::output(void)
         tailsitter_motors->set_min_throttle(0.0);
     }
 
-    tilt_left = 0.0f;
-    tilt_right = 0.0f;
```

Comment:
Agreed. The fact that we were nulling tilt here basically creates a notional separation between its previous usages.